src/dialog-flow-schema.js

```diff
@@ -23,7 +23,7 @@ class dialogFlow {
   }
 
   static get VALID_LOCALES() {
-    return ['en-US','en-GB', 'de-DE'];
+    return ['en-AU', 'en-US','en-GB', 'de-DE'];
```

Comment:
Could you also add the rest of the locales valid for Alexa and Google Assistant ? `en-CA`, `en-IN`, `fr-FR`

Reply:
@BruceLampson you missed the locales for the dialogflow file

Reply:
@omenocal are we looking at matching the dialog flow schema valid locales to the ones in `src/alexa-schema.js` or actually adding the supported locales? I'm asking because I found an article with a list of supported locales for **Actions on Google** and wondering if I should add all of these https://developers.google.com/actions/localization/languages-locales

Reply:
@BruceLampson I first thought only matching both platform's locales. But you've brought up a good point. This repo is supposed to be used by anyone, so we should support all locales if this library is meant to help other developers.